pkgs/tools/networking/haguichi/update.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts
+
+set -euo pipefail
+
+nixpkgs="$(git rev-parse --show-toplevel)"
+
+oldVersion=$(nix-instantiate --eval -E "(import \"$nixpkgs\" { config = {}; overlays = []; }).haguichi.version" | tr -d '"')
+version=$(curl -s https://api.github.com/repos/ztefn/haguichi/releases | jq '.[] | select(.target_commitish!="elementary") | .tag_name' --raw-output | sort --version-sort --reverse | head -n1)
+
+if [[ $version == $oldVersion ]]; then
+  echo "Already at latest version $version"
+  exit 0
+fi
```

Comment:
`update-source-version` seems to do this check itself.

Reply:
You're right, removed the custom check.

https://github.com/NixOS/nixpkgs/blob/5a350a8f31bb7ef0c6e79aea3795a890cf7743d4/pkgs/common-updater/scripts/update-source-version#L142-L148